src/defaultProps.js

```diff
@@ -23,7 +23,7 @@ export default {
   defaultSorting: [],
   showFilters: false,
   defaultFiltering: [],
-  defaultFilterMethod: (filter, row, column) => (row[filter.id] == filter.value),
+  defaultFilterMethod: (filter, row, column) => (row[filter.id] === filter.value),
```

Comment:
This actually doesn't behave how I think it should for columns with numbers. I suppose I could cast the row's value to a string, but that didn't seem as clean as fuzzy equality. I understand the need to pass the linter though for the CI.